pkg/settings/setting.go

```diff
@@ -113,6 +113,9 @@ var (
 
 	FleetMinVersion          = NewSetting("fleet-min-version", "")
 	RancherWebhookMinVersion = NewSetting("rancher-webhook-min-version", "")
+	AKSOperatorMinVersion    = NewSetting("aks-operator-min-version", "")
+	EKSOperatorMinVersion    = NewSetting("eks-operator-min-version", "")
+	GKEOperatorMinVersion    = NewSetting("gke-operator-min-version", "")
```

Comment:
Also curious about this part, would we need to update every time ? Older version used `LatestAvailableTemplateVersion` to grab newest one. 

edit: 
Oh I missed your comment a few days back: 

> There is a minimal version setting for each operator so that we make sure for upgrading setup, if the current operator version meets the min version of the setting, it will not try to upgrade to the latest version. The behavior is defined in here #30934. This is to prevent that we don't intentionally upgrade user's operator if min version is satisfied.

> But if there is a fresh install it will always try to get the latest version, this is not a one-operator-version-per-rancher-version model(2.6.0 -> 1.1.0, 2.6.1 -> 1.1.1), as it will always try to install the latest version of the chart in feature charts(so 2.6.x -> latest version of operator). But if there is a newer rancher-operator version that won't work with old rancher version, we can use rancher min/max version on catalog to filter out the latest charts.

So to confirm we can use min/max to filter this ? Ideally we would always want the KEv2 operators to have the latest compatible chart for their version line. 
So 2.6.4 gets latest release in release-2.6 branch that is compatible with 2.6.4 (maybe newer one exists but min is 2.6.5). 
If we have to specify that in env vars that is possible, but not the best process. I'm surprised fleet uses this model, I know we don't want the _actual_ latest chart, but latest that uses min/max would be fine no ?

Reply:
@cbron Yes, min/max can also be used in feature chart, It will fetch the latest chart in the release branch(release-2.6) that is compatible with the min/max setting. The only difference in this model is that for existing setup, we don't upgrade to the latest compatiable model if the min version is satified. So user won't get an upgrade unless they upgrade rancher, which changes min version settings. Fleet uses this model and I was told to apply it to kev2 charts. But if that's not the case I can remove this min version so it always install the latest compatible version which is the same as how it currently works.

Reply:
Seems like you've done that above, do you still need these settings ? 
If its because if you set these you get the "fleet model" there should be comment saying thats how it works, as its somewhat counterintuitive.

Reply:
@thedadams @rmweir if we remove these we will have to set min/max on every feature chart version, otherwise every newer release that goes into the release branches will get pulled into old ranchers, which could crash things. Whats your preferred route, updating these all the time or updating min/max in the chart ? 
The chart is better way IMO, but with feature charts its hard to adjust old ones, you have to replace tarball. So we'll either have to go route of: 
a) set a max on every release and continuously update that to bump it
b) not set a max and then add one when there is a new release.
Either ways are error prone. Which then leads me back to using this min version, as it would be obvious if we forgot it.

Reply:
@StrongMonkey I think we should remove these and then we will just put a min on every feature chart version that matches rancher version.

Reply:
@cbron Settings are also removed.